### PR TITLE
RET-3917

### PR DESCRIPTION
**Pull request by hsjhita**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-3917

### Change description ###

Integrate ET into EM Stitching and Bundling tools

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by btensay — APPROVED** [review]

File: definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+    "FieldType": "FixedList",
+    "ElementLabel": "How would you like to have the image rendered?",
+    "FieldTypeParameter": "imageRendering",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "DocumentImage",
+    "ListElementCode": "imageRenderingLocation ",
+    "FieldType": "FixedList",
+    "ElementLabel": "Where would you like to embed the image in stitched document?",
```
Comment (JackReevies) [inline definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json:132]: in the stitched document?

File: definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+    "ID": "DocumentImage",
+    "ListElementCode": "imageRendering",
+    "FieldType": "FixedList",
+    "ElementLabel": "How would you like to have the image rendered?",
+    "FieldTypeParameter": "imageRendering",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "DocumentImage",
+    "ListElementCode": "imageRenderingLocation ",
```
Comment (JackReevies) [inline definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json:130]: extra space at the end here

File: definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+    "FieldType": "FixedList",
+    "ElementLabel": "Where would you like to embed the image in stitched document?",
+    "FieldTypeParameter": "imageRenderingLocation ",
```
Comment (JackReevies) [inline definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json:133]: extra space at the end here

File: definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "DocumentImage",
+    "ListElementCode": "docmosisAssetId",
+    "FieldType": "Text",
+    "ElementLabel": "The filename of the Image you wish to embed",
```
Comment (JackReevies) [inline definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json:140]: capital I on image?

File: definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+    "ListElementCode": "hasFolderCoversheets",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Should this bundle’s folders have a coversheet?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Bundle",
+    "ListElementCode": "stitchingFailureMessage",
+    "FieldType": "Text",
+    "ElementLabel": "Error from Stiching service",
```
Comment (JackReevies) [inline definitions/json/ComplexTypes/ComplexTypes-DCF2-nonprod.json:247]: I imagine a callback is going to overwrite this anyway but spelling mistake in stitching

**Review by JackReevies — CHANGES_REQUESTED** [review]

**Review by JackReevies — APPROVED** [review]

**Pull request merged**